### PR TITLE
Fix get deployment status issues

### PR DESCRIPTION
**Pull request by Flanker32**
- Fix getting deployment status timeout wording
- Extend deploy status pending time to 180s

### CODE REVIEW

File: azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/DeployMojo.java [hunk 1]
```diff
@@ -49,16 +49,16 @@
...
     protected static final String PROJECT_SKIP = "Packaging type is pom, taking no actions.";
     protected static final String PROJECT_NO_CONFIGURATION = "Configuration does not exist, taking no actions.";
     protected static final String PROJECT_NOT_SUPPORT = "`azure-spring-cloud:deploy` does not support maven project with " +
             "packaging %s, only jar is supported";
     protected static final String GET_APP_URL_SUCCESSFULLY = "Application url : %s";
     protected static final String GET_APP_URL_FAIL = "Fail to get application url";
     protected static final String GET_APP_URL_FAIL_WITH_TIMEOUT = "Fail to get application url in %d s";
-    protected static final String GET_DEPLOYMENT_STATUS_TIMEOUT = "Spring Cloud Service fail to start up in %d seconds, " +
-            "please check the status in portal.";
+    protected static final String GET_DEPLOYMENT_STATUS_TIMEOUT = "Deployment succeed but the app is still starting " +
```
Comment (andxu) [inline azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/DeployMojo.java]: Deploy succeeded

File: azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/DeployMojo.java [hunk 2]
```diff
@@ -49,16 +49,16 @@
...
+            "after %d seconds, you can check app status from Azure Portal.";
```
Comment (andxu) [inline azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/DeployMojo.java]: `the` app status

Comment (Flanker32) [reply azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/DeployMojo.java]: Fixed

File: azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/DeployMojo.java (hunk 1 above)
Comment (Flanker32) [reply azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/DeployMojo.java]: Fixed

**Pull request merged**